@uportal/sidebar-nav/src/components/SidebarNav.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="sidebar-nav" v-if="linkDataUrl">
+    <div class="sidebar-nav__accordions">
+      <accordion
+        v-for="(accordionItem, index) in linkData.topics"
+        :topic="accordionItem"
+        :key="index"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import Accordion from './Accordion';
+
+export default {
+  name: 'SidebarNav',
+  props: {
+    linkDataUrl: {
+      type: String,
+      default: null
+    }
+  },
+  data() {
+    return {
+      linkData: null
+    };
+  },
+  components: {
+    Accordion
+  },
+  mounted() {
+      if (this.linkDataUrl) {
+          fetch(this.linkDataUrl)
+              .then(r => r.json())
+              .then(json => {
+                  this.linkData = json;
+              });
+      }
+  }
+};
+</script>
+
+<style lang="scss">
+a,
+a:visited {
+  text-decoration: none;
+}
```

Comment:
without `scoped` these styles will bleed out of the component on to the page in [browsers that don't support shadow dom](https://caniuse.com/#feat=shadowdomv1).
is that the intended/expected behavior?

Reply:
You're right, good catch!